frontend/services/index.ts

```diff
@@ -27,6 +28,9 @@ export const sendRequest = async (
       },
     });
 
+    if (returnRaw) {
+      return response;
+    }
```

Comment:
just for my own understanding: why/how did this work before?

Reply:
The method was extracting only the `data` field and returning that to the caller (at the next line 34). That's fine for 99% of the usage where the caller doesn't care about the headers. IMO we're overdue to revamp `sendRequest` (which has always been a just thin wrapper around axios) but I didn't want to get into the weeds at this stage.